### PR TITLE
Porting System.security.cryptography.encryption.ecdsa

### PR DESCRIPTION
**Pull request by venkat-raman251**

### CODE REVIEW

**Comment by venkat-raman251** [comment]
@weshaggard  : Please have a look.

**Comment by weshaggard** [comment]
cc @bartonjs

File: src/Common/src/Internal/NativeCrypto/Cng.cs [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+
+using System;
+using System.Text;
+using System.Diagnostics;
+using System.Diagnostics.Contracts;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Runtime.InteropServices;
+
+namespace Internal.NativeCrypto
```
Comment (weshaggard) [inline src/Common/src/Internal/NativeCrypto/Cng.cs]: This isn't following the interop guidelines.

**Comment by venkat-raman251** [comment]
Was a prototype project. Closing PR

**Pull request closed without merging**